src/Rules/Classes/AllowedSubTypesRule.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Classes;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Node\InClassNode;
+use PHPStan\Rules\Rule;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\Type\ObjectType;
+use function array_values;
+use function sprintf;
+
+/**
+ * @implements Rule<InClassNode>
+ */
+class AllowedSubTypesRule implements Rule
```

Comment:
This rule isn't registered. Level 0 would be fine 👍

Reply:
Oops, added, thanks.